docs/blog/2017-12-20-introducing-the-gatsby-ux-research-program/index.md

```diff
@@ -70,17 +70,23 @@ The second most common pain point is GraphQL. About half of our interviewees sai
->
-> – David Luhr, UT, UX
+– Alec Lomas, AZ, front-end dev and design
+</pullquote>
 
-> If you ever decide to sell to consumers, I really think that the speed / responsiveness of Gatsby is a really good selling point. Like when I tried to sell my idea of building the new site in Gatsby (which would take a little longer compared to building it in Squarespace) to my husband, I simply showed him how quick it is to navigate around a Gatsby site and he was instantly sold. The explicit awareness that website speed matters a lot is currently kinda limited to Silicon Valley etc, but the benefits are definitely apparent as well to the average consumer. Pair that with stats around how speed increases conversions and it'd definitely be a pretty appealing offer to people.
->
-> – Juliette Pichot, Germany, web developer
+<pullquote>
+I’ve played around with static sites in Jekyll in the past, but I’ve always hit a pain point with how much functionality I could build into the different includes and partials and things that were starting to become popular with different templating languages. So I’ve always loved React for all the encapsulation of style and functionality. And components--but I never really knew how to use it. And Gatsby has been that first opportunity where everything comes together, everything can be in one system. My background is really design focused, so I think of a component library along those lines. Until Gatsby, I never really had a way to see how I could use that across all of our pages instead of relying on a content management library that was kind of slow to develop in.
+
+– David Luhr, UT, UX
+</pullquote>
+
+<pullquote>
+If you ever decide to sell to consumers, I really think that the speed / responsiveness of Gatsby is a really good selling point. Like when I tried to sell my idea of building the new site in Gatsby (which would take a little longer compared to building it in Squarespace) to my husband, I simply showed him how quick it is to navigate around a Gatsby site and he was instantly sold. The explicit awareness that website speed matters a lot is currently kinda limited to Silicon Valley etc, but the benefits are definitely apparent as well to the average consumer. Pair that with stats around how speed increases conversions and it'd definitely be a pretty appealing offer to people.
+
+– Juliette Pichot, Germany, web developer
```

Comment:
@greglobinski @amberleyromo we'll probably want some way to cite (via a prop/attribute) authors, correct?

Reply:
I think it's ideal to have since we have such citings over a couple of posts.